src/site/stages/build/drupal/page.js

```diff
@@ -165,6 +165,35 @@ function generateBreadCrumbs(pathString) {
   return entityUrlObj;
 }
 
+// Generate breadcrumbs from GraphQL breadcrumbs
+function generateDrupalBreadCrumbs(entityUrl) {
+  const entityUrlObj = {
+    breadcrumb: [],
+    path: entityUrl.path,
+  };
+
+  if (typeof entityUrl.breadcrumb !== 'undefined') {
+    for (const value of entityUrl.breadcrumb) {
+      if (value.text) {
+        const dehandlized =
+          value === 'pittsburgh-health-care'
+            ? 'VA Pittsburgh health care'
+            : value.text;
+
```

Comment:
Is there no way to do this in the CMS, or is this just a temporary solution? Asking because I imagine eventually we'll  have more than just Pittsburgh so we'll find ourselves doing this quite a bit.

Reply:
@acabouet can you share any insight? It just seems like a lot of additional logic for deriving a value that seems like should be stored in the CMS. At some point, we may find that we need to update that breadcrumb (for example, if at some point we want to omit `VA` or use title case) and this could also cause some confusion there.

Reply:
Jonathan's understanding is correct, @ncksllvn. A number of the pittsburgh health care pages only exist in the front end, they don't have corresponding entities in drupal (the news, stories, and events listing pages are each examples of this) and so we have to build their breadcrumbs from their path, but the path doesn't have the VA prefix. The only way to fix this on the drupal side would be to change the path from pittsburgh-health-care to va-pittsburgh-health-care but that would obviously affect the entirety of this subsite. This is a short term fix until we figure out a long term way to address this.

Reply:
Oh interesting, thanks @acabouet!

Reply:
Having those pages only exist only on the front end has a few negative side effects, like taking away control of the url from content editors and the pages not being previewable. A good long term solution might be to create entity types for the in Drupal, even if the content you can change is minimal.

Reply:
@jonbot  True, but it would provide control of that page route to the content team, which I think is ideal.

@jonbot Can you see my comment [here](https://github.com/department-of-veterans-affairs/vets-website/pull/10298#discussion_r296901766)?

Reply:
@jonbot I don't think we'd have to maintain content in two places, I think the front end would pull the path and any other content from Drupal, like it does for most pages.